index.js

```diff
@@ -214,5 +220,54 @@ module.exports = {
       'space-unary-ops': 'error',
       'unicode-bom': 'error',
 
+      'arrow-spacing': [ 'error', { 'before': true, 'after': true } ],
    },
+
+   'overrides': [
+      {
+         'files': [ '**/*.ts' ],
+         'parser': 'typescript-eslint-parser',
+         'parserOptions': {
+            'sourceType': 'module',
+         },
+         'rules': {
+            // TODO: figure out how to fix no-undef.
+            // Currently, no-undef causes false positives for TypeScript Class Properties.
+            // With TypeScript only code, this rule can safely be disabled, because
```

Comment:
```suggestion
            // With TypeScript only code this rule can safely be disabled because
```

Reply:
Wow... @jthomerson asking me to **remove** commas. I never thought I'd see the day. 😄